Dockerfile

```diff
@@ -38,10 +39,27 @@ RUN wget -O /tmp/spark-${SPARK_VER}-bin-hadoop${HADOOP_VER}.tgz https://archive.
     rm -rf /tmp/spark-${SPARK_VER}-bin-hadoop${HADOOP_VER}.tgz && \
     mv /spark-${SPARK_VER}-bin-hadoop${HADOOP_VER} ${SPARK_HOME}
 
+# The current R version on EPEL is install as a requirement of JAP and is not 
+# frozen. Environment variables R_LIBS_SITE and R_LIBS_USER paths are included
+# in paths searched by R for library packages. Only directories which exist
+# will be included.
+ENV R_LIBS_SITE_ROOT /opt/R-libs/site
+ENV R_LIBS_SITE_USER /opt/R-libs/user
+ENV R_LIBS_SITE $R_LIBS_SITE_USER:$R_LIBS_SITE_ROOT
+# Install core libraries as root user
+RUN mkdir -p $R_LIBS_SITE_ROOT && \
+    R -q -e "install.packages(c('devtools', 'dplyr', 'IRdisplay', 'knitr', 'magrittr', 'packrat'), repos='https://cloud.r-project.org/')" && \
```

Comment:
Why is this a different repo to below?

Reply:
I Missed those, will fix them both to be the official site (`cloud.r-project.org`).